nav2_smac_planner/src/smac_planner_hybrid.cpp

```diff
@@ -616,6 +629,20 @@ SmacPlannerHybrid::dynamicParametersCallback(std::vector<rclcpp::Parameter> para
             "valid options are MOORE, VON_NEUMANN, DUBIN, REEDS_SHEPP.",
             _motion_model_for_search.c_str());
         }
+      } else if (name == _name + ".goal_heading") {
+        std::string goal_heading_type = parameter.as_string();
+        _goal_heading = fromStringToGH(goal_heading_type);
```

Comment:
don't update unless not unknown - as an invalid configuration, we don't want to update the class member